changhyun/app.js

```diff
@@ -33,4 +33,18 @@ app.get("/ping", function (req, res, next) {
   return res.status(200).json({ message : "pong" });
 })
 
+app.post('/create', async (req, res) => {
+	const { name, email, profile_image, password } = req.body
+	await appDataSource.query(
+		`INSERT INTO users(
+      name,email,
+      profile_image,
+      password
+      ) VALUES (?, ?, ?, ?);
+      `,
+      [ name, email, profile_image, password ]
+    ); 
```

Comment:
SQL문의 indentation 수정 부탁드립니다~
```suggestion
	await appDataSource.query(
	    `INSERT INTO users(
                  name,
                  email,
                  profile_image,
                  password
              ) VALUES (?, ?, ?, ?);
          `,
          [ name, email, profile_image, password ]
        ); 
```

- SQL 문법도 가독성 좋게 만들어주는 것이 중요합니다. 회사마다 컨벤션이 다르겠지만, 위코드에서는 아래와 같은 SQL 문법의 컨벤션을 정하고 있습니다.
- 지금은 위코드에서 사용하고 있는 컨벤션을 지켜서 문법을 작성해주세요. 하지만 이는 회사마다 다르기 때문에, 정답이 아님을 반드시 인지해주세요.

Reply:
<img width="581" alt="Screen Shot 2022-11-04 at 2 32 53 PM" src="https://user-images.githubusercontent.com/100187797/199897957-bb320e52-d7d6-40d9-9b0c-d112d55ce13d.png">

이런식으로 저장 했었는데 보내지면서 오류가 난것 같습니다.